imageroot/systemd/user/mariadb-app.service

```diff
@@ -16,12 +17,11 @@ ExecStart=/usr/bin/podman run --conmon-pidfile %t/mariadb-app.pid \
     -v mysql-data:/var/lib/mysql/:Z -v mysql-conf.d:/etc/mysql/conf.d:Z \
     -v sql:/docker-entrypoint-initdb.d:Z \
     -v %S/scripts/create_admin_of_phpmyadmin.sql:/docker-entrypoint-initdb.d/create_admin.sql:Z \
-    -e MARIADB_ROOT_PASSWORD_FILE=/run/secrets/root.secret -e MARIADB_DATABASE=phpmyadmin \
-    -e MARIADB_USER=phpmyadmin -e MARIADB_PASSWORD_FILE=/run/secrets/phpmyadmin.secret \
-    --secret root.secret --secret phpmyadmin.secret \
+    -e MARIADB_ROOT_PASSWORD=${MARIADB_ROOT_PASSWORD} -e MARIADB_DATABASE=phpmyadmin \
+    -e MARIADB_USER=phpmyadmin -e MARIADB_PASSWORD=${MARIADB_PASSWORD} \
     ${MARIADB_IMAGE} \
     --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
-ExecStartPost=/usr/bin/podman exec  mariadb-app /bin/bash -c 'printf "[client] \npassword=$(cat /run/secrets/root.secret)" > /root/.my.cnf'
+ExecStartPost=/usr/bin/podman exec  mariadb-app /bin/bash -c 'printf "[client] \npassword=${MARIADB_ROOT_PASSWORD}" > /root/.my.cnf'
```

Comment:
As alternative, if the mariadb image allows it, put an executable bash script in `docker-entrypoint-initdb.d` that configures .my.cnf properly

Reply:
Sorry many attempts but somethings occurs badly, the mysql alteration does not occur anymore when I try to move the script to this path. Back to the former version

Reply:
Ok let's merge it as is for now